res/raw/webxdc.js

```diff
@@ -70,16 +68,14 @@ window.webxdc = (() => {
       if (message.file) {
```

Comment:
wondering if the type script annotation `/** @type {{file_name: string, file_message: string} | null} */` belongs to `message.file` and if it is correct - we do nowhere have the parameters or variables named `file_name`  or `file_message`